pkg/kubernetes/utils/utils.go

```diff
@@ -401,12 +401,24 @@ func GetKubeletCommand(ctx context.Context, podExecutor pod.PodExecutor) (string
 }
 
 // GetContainerCommand returns the used container command
-func GetContainerCommand(pod corev1.Pod, containerName string) (string, error) {
-	container, found := GetContainerFromPod(&pod, containerName)
-	if !found {
-		return "", fmt.Errorf("pod does not contain %s container", containerName)
+func GetContainerCommand(pod corev1.Pod, containerNames ...string) (string, error) {
+	var (
+		container corev1.Container
+		found     = false
+	)
+
+	for _, containerName := range containerNames {
+		var foundContainer corev1.Container
+		foundContainer, found = GetContainerFromPod(&pod, containerName)
+		if found {
+			container = foundContainer
+			break
+		}
 	}
 
+	if !found {
+		return "", fmt.Errorf("pod does not contain any of the containers specified in the provided list: %s", containerNames)
+	}
 	rawCommand := strings.Join(append(container.Command, container.Args...), " ")
 	return rawCommand, nil
 }
```

Comment:
```suggestion
// The first container name that exists in the Pod is selected
func GetContainerCommand(pod corev1.Pod, containerNames ...string) (string, error) {
	for _, containerName := range containerNames {
		if container, found := GetContainerFromPod(&pod, containerName); found {
			rawCommand := strings.Join(append(container.Command, container.Args...), " ")
			return rawCommand, nil
		}
	}

	return "", fmt.Errorf("pod does not contain container with name in %v", containerNames)
}
```
Let's make the function simpler while keeping the same functionality.

Reply:
Please use the suggestion. The full comment is missing, the `if` clause can be on the line where the variables are declared and the error message is different.